### ISSUE TITLE
Ignore accents in the search box

### ISSUE THREAD

**Issue by baudav** [Kunzisoft/KeePassDX#945]
**Is your feature request related to a problem? Please describe.**
as much there is probably no problem in English, it is not the case in French and other language using accents or cedillas. When searching for an entry, it may be necessary to perform several searches. For example for 'crédit' you will have to search for 'credit' but also 'crédit'. For 'François', it will be necessary to search for francois and françois. (although sometimes the username does not contain the accent and still pops out as a result)

**Describe the solution you'd like**
Just as the case is ignored, it would be nice if accents and cedilla were ignored in the search.

**Comment by J-Jamet** [comment]
Unfortunately, the function that allow this functionality are too slow. So the functionality is rollback for the moment.

**Comment by Vstory** [comment]
>Unfortunately, the function that allow this functionality are too slow. So the functionality is rollback for the moment.

Is it possible to pop up a search dialog box separately and use this function as an option.

I have an idea to provide advanced search options in the settings.
When the advanced search is enabled, after clicking the search icon, a search dialog box will pop up, which contains some more advanced search commands. Advanced commands can include the above functions, as well as ignore case, etc.

This not only keeps it simple, but also satisfies users who need more detailed searches.They should be able to ignore the performance issues of advanced search.

**Comment by J-Jamet** [comment]
Yes it is planned, it will contain a lot of options but it is not a simple function to add and requires several hours of development and testing so I will do it in another version.

**Comment by cbiere** [comment]
As far as I remember, the Unicode standard has built-in recommendations and procedures for this. So by taking the classification of characters into consideration, you can transform a string into a basic variant that is appropriate for pattern matching respectively searching without having to know the specifics of the used language or characters.
